src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -31,6 +32,16 @@ export const ConversationSpace: React.FunctionComponent = () => {
 
   useExperiences(venue?.name);
 
+  const parentVenueId = venue?.parentId;
+  const parentVenue = useSelector(parentVenueSelector);
+
+  // @debt This logic is a copy paste from NavBar. Move that into a separate Back button component
+  const backToParentVenue = useCallback(() => {
+    if (!parentVenueId) return;
+
+    openUrl(venueInsideUrl(parentVenueId));
+  }, [parentVenueId]);
+
```

Comment:
could you please use `useRelativeVenues` here? The example is in NavBar

Reply:
fixed in fae8270fb609124c5b2b41895cad7dced8bf0718